odmlui/treemodel/GenericIter.py

```diff
@@ -41,7 +41,7 @@ def get_next(self):
         returns a new Iter object for the next element in this multivalue list
         or None
         """
-        obj = next(self._obj)
+        obj = self._obj.__next__()
```

Comment:
Was this changes to accommodate Python 2? I know `next()` is available in 2.6+. Are we supporting 2.5?
Calling `__next__()` is kinda ugly.